### ISSUE TITLE
How to interpret output from dismo::evaluate function

### ISSUE THREAD

**Issue by TomNash** [TomNash/vaccine-project#7]
From the `?dismo::maxent` example we have the output below for the `evaluate` function, the documentation doesn't provide much insight on the variables. I see there's a TPR, TNR, FPR, and FNR confusion matrix, but it's done on a per instance basis (999 x 4) matrix. Not sure what `0.154112` signifies for the `max TPR+TNR` output. Not sure what's useful or how to get useful information from it. This is pretty much the last step.

```
> e1
class          : ModelEvaluation 
n presences    : 23 
n absences     : 999 
AUC            : 0.879053 
cor            : 0.2468145 
max TPR+TNR at : 0.154112 
```

Plots from the example don't provide much insight either:

`plot(e1, 'TPR')`
![image](https://cloud.githubusercontent.com/assets/8193065/14445419/d2e448a8-001a-11e6-9463-97b4d5e44cac.png)

`density(e1)`
![image](https://cloud.githubusercontent.com/assets/8193065/14445438/f2193396-001a-11e6-9367-6ca8d381acdd.png)

**Comment by dmcglinn** [comment]
I think those outputs are as follows

FPR is false positive rate - proportion of times model said an outbreak would be present but it was not 
TPR is true positive rate - prop outbreak occurred and model predicted it 
FNR is false negative rate - prop outbreak did occur but model said it wouldn't. 
TNR is true negative rate - prop outbreak did not occur and model said it would not occur.

There should be a threshold level to set that was needed to convert the model's likelihood output into a binary prediction the logical option would be 0.5 if the output us bounded on 0-1.

It think the max TPR+TNR is a measure of how good the model does it's like the proportion the model is correct. I'm not sure why max is used though.

If you want to dig into this better I would look at the Phillips et al paper that developed the Java code you are using.

Dan

**Comment by TomNash** [comment]
I figured that's what the FPR, TPR, etc. meant. There is another function to be run on the evaluate, `threshold` that gives this:

```
> threshold(e1)
               kappa spec_sens no_omission prevalence equal_sens_spec sensitivity
thresholds 0.7363298 0.1445414  0.03797142 0.02293959       0.3120633   0.1988572
```

I found [this paper referenced in the documentation](http://www.dpi.inpe.br/referata/arq/14.Camilo/fielding_Bell_1997.pdf) that explains some of the variables, Kappa is apparently very informative of the overall strength as it uses the entire confusion matrix (starting pg 42, K < 0.4 is poor, 0.4 < K < 0.75 is good, K > 0.75 is excellent). I'm thinking add a new tab which includes the information from the `threshold` output?

EDIT: Oh I see, this is identifying cutoff points based on each of these statistics? So 0.5 is the default, but if you set it to 0.7363 (as below), this would maximize Kappa in binary prediction? So the 0.7363 isn't Kappa, but the threshold which obtains maximum Kappa.

```
> t(threshold(e1))
                thresholds
kappa           0.73632981
spec_sens       0.14454137
no_omission     0.03797142
prevalence      0.02293959
equal_sens_spec 0.31206329
sensitivity     0.19885716
```

```
> e1@kappa[which(e1@t==threshold(e1)$kappa)]
[1] 0.1643132
> max(e1@kappa)
[1] 0.1643132
```
